pallets/registries/src/tests.rs

```diff
@@ -1379,3 +1379,656 @@ fn add_delegator_should_fail_if_registry_delegates_limit_exceeded() {
 		);
 	});
 }
+
+#[test]
+fn remove_delegate_should_fail_if_admin_authorization_is_not_found() {
+	let creator = ACCOUNT_00;
+	let delegate = ACCOUNT_01;
+	let registry = [2u8; 256].to_vec();
+
+	let raw_blob = [2u8; 256].to_vec();
+	let blob: RegistryBlobOf<Test> = BoundedVec::try_from(raw_blob)
+		.expect("Test blob should fit into the expected input length for the test runtime.");
+
+	let registry_digest = <Test as frame_system::Config>::Hashing::hash(&registry.encode()[..]);
+
+	let id_digest = <Test as frame_system::Config>::Hashing::hash(
+		&[&registry_digest.encode()[..], &creator.encode()[..]].concat()[..],
+	);
+
+	let registry_id: RegistryIdOf = generate_registry_id::<Test>(&id_digest);
+
+	let auth_id_digest = <Test as frame_system::Config>::Hashing::hash(
+		&[&registry_id.encode()[..], &creator.encode()[..], &creator.encode()[..]].concat()[..],
+	);
+
+	let non_existent_auth_id_digest =
+		<Test as frame_system::Config>::Hashing::hash(&registry_id.encode()[..]);
+
+	let authorization_id: AuthorizationIdOf = generate_authorization_id::<Test>(&auth_id_digest);
+
+	let non_existent_authorization_id: AuthorizationIdOf =
+		generate_authorization_id::<Test>(&non_existent_auth_id_digest);
+
+	let delegate_auth_id_digest = <Test as frame_system::Config>::Hashing::hash(
+		&[&registry_id.encode()[..], &delegate.encode()[..], &creator.encode()[..]].concat()[..],
+	);
+
+	let delegate_authorization_id: AuthorizationIdOf =
+		generate_authorization_id::<Test>(&delegate_auth_id_digest);
+
+	let raw_schema = [2u8; 256].to_vec();
+	let schema: InputSchemaOf<Test> = BoundedVec::try_from(raw_schema)
+		.expect("Test Schema should fit into the expected input length for the test runtime.");
+	let _digest: SchemaHashOf<Test> = <Test as frame_system::Config>::Hashing::hash(&schema[..]);
+	let schema_id_digest = <Test as frame_system::Config>::Hashing::hash(&schema.encode()[..]);
+	let schema_id: SchemaIdOf = generate_schema_id::<Test>(&schema_id_digest);
+
+	new_test_ext().execute_with(|| {
+		assert_ok!(Registries::create(
```

Comment:
Here before registry create operation there is a new need that a namespace should be created. That namespace authorisation id must be passed to all `Registry` based operations.
Please refer updated test.rs file.